packages/lesswrong/components/ea-forum/wrapped/EAForumWrappedPage.tsx

```diff
@@ -1639,8 +1689,8 @@ const EAForumWrapped2023Page = ({classes}: {classes: ClassesType}) => {
         
         <AnalyticsContext pageSectionContext="top">
           <section className={classes.section}>
-            <h1 className={classes.heading1}>Your 2023 Wrapped</h1>
-            {currentUser.wrapped2023Viewed &&
+            <h1 className={classes.heading1}>Your {year} Wrapped</h1>
+            {(year !== 2023 || currentUser.wrapped2023Viewed) &&
               <button className={classNames(classes.summaryLinkWrapper, classes.skipToSummaryBtn)} onClick={skipToSummary}>
                 Skip to summary
```

Comment:
The changes in this file are to make it functional for 2022, including making the "Skip to summary" button always visible for 2022